database/drivers/mysql/parse.go

```diff
@@ -126,16 +126,29 @@ func parse(log *log.Logger, conn string, schemaNames []string, filterTables func
 
 		schemaIndex, ok := indexes[s.TableSchema]
 		if !ok {
-			schemaIndex = make(map[string]map[string][]*database.Column)
+			schemaIndex = make(map[string][]*database.Index)
 			indexes[s.TableSchema] = schemaIndex
 		}
 
 		tableIndex, ok := schemaIndex[s.TableName]
 		if !ok {
-			tableIndex = make(map[string][]*database.Column)
+			tableIndex = make([]*database.Index, 0)
```

Comment:
you can drop the ,0 here, since that's the default (and I think go vet or go lint will complain at you)

Reply:
I had removed the 0 and `go vet` complained, but I didn't spend too much time on it. I'll take another look.

Reply:
I have verified that the length parameter is required. The exception being `[]*database.Index{}` syntax.